.cicd/scripts/srw_test.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
It would help to have a comment header at the top to say what this script does (check that there is a non-zero sized executable for the weather model, I think, and return with a zero or non-zero exit status accordingly).